app/controllers/consumption/testOnly/ListRetrievalController.scala

```diff
@@ -0,0 +1,115 @@
+
+  def get(listName: ListName): Action[AnyContent] =
+    Action {
+      listName.listName match {
+        case "CustomsOffices" =>
+          Ok(Json.obj("data" -> Json.toJson(listRetrievalService.getCustomsOffice)))
+        case "CountryCodesFullList" =>
+          Ok(Json.obj("data" -> Json.toJson(listRetrievalService.getCountryCodesFullList)))
+        case "CountryCodesCommonTransit" =>
+          Ok(Json.obj("data" -> Json.toJson(listRetrievalService.getCountryCodesCommonTransit)))
+        case "CountryCodesCTC" =>
+          Ok(Json.obj("data" -> Json.toJson(listRetrievalService.getCountryCodesCTC)))
+        case "CountryCodesCommunity" =>
+          Ok(Json.obj("data" -> Json.toJson(listRetrievalService.getCountryCodesCommunity)))
+        case "CountryCodesForAddress" =>
+          Ok(Json.obj("data" -> Json.toJson(listRetrievalService.getCountryCodesForAddress)))
+        case "CountryCustomsSecurityAgreementArea" =>
+          Ok(Json.obj("data" -> Json.toJson(listRetrievalService.getCountryCustomsSecurityAgreementArea)))
+        case "CountryAddressPostcodeBased" =>
+          Ok(Json.obj("data" -> Json.toJson(listRetrievalService.getCountryAddressPostcodeBased)))
+        case "CountryWithoutZip" =>
+          Ok(Json.obj("data" -> Json.toJson(listRetrievalService.getCountryWithoutZip)))
+        case "UnLocodeExtended" =>
+          Ok(Json.obj("data" -> Json.toJson(listRetrievalService.getUnLocodeExtended)))
+        case "Nationality" =>
+          Ok(Json.obj("data" -> Json.toJson(listRetrievalService.getNationality)))
+        case "PreviousDocumentType" =>
+          Ok(Json.obj("data" -> Json.toJson(listRetrievalService.getPreviousDocumentType)))
+        case "SupportingDocumentType" =>
+          Ok(Json.obj("data" -> Json.toJson(listRetrievalService.getSupportingDocumentType)))
+        case "TransportDocumentType" =>
+          Ok(Json.obj("data" -> Json.toJson(listRetrievalService.getTransportDocumentType)))
+        case "KindOfPackages" =>
+          Ok(Json.obj("data" -> Json.toJson(listRetrievalService.getKindOfPackages)))
+        case "KindOfPackagesBulk" =>
+          Ok(Json.obj("data" -> Json.toJson(listRetrievalService.getKindOfPackagesBulk)))
+        case "KindOfPackagesUnpacked" =>
+          Ok(Json.obj("data" -> Json.toJson(listRetrievalService.getKindOfPackagesUnpacked)))
+        case "AdditionalReference" =>
+          Ok(Json.obj("data" -> Json.toJson(listRetrievalService.getAdditionalReference)))
+        case "AdditionalInformation" =>
+          Ok(Json.obj("data" -> Json.toJson(listRetrievalService.getAdditionalInformation)))
+        case "Unit" =>
+          Ok(Json.obj("data" -> Json.toJson(listRetrievalService.getUnit)))
+        case "CurrencyCodes" =>
+          Ok(Json.obj("data" -> Json.toJson(listRetrievalService.getCurrencyCodes)))
+        case "ControlType" =>
+          Ok(Json.obj("data" -> Json.toJson(listRetrievalService.getControlType)))
+        case _ => NotFound
+      }
```

Comment:
Again, very minor but a partial function could help remove some of the duplication here
```
val dataRetrieval: PartialFunction[String, JsValue] = {
        case "CustomsOffices" =>
          Json.toJson(listRetrievalService.getCustomsOffice)
        case "CountryCodesFullList" =>
          Json.toJson(listRetrievalService.getCountryCodesFullList)
        case "CountryCodesCommonTransit" =>
          Json.toJson(listRetrievalService.getCountryCodesCommonTransit)
        ...
      }

      dataRetrieval
        .lift(listName.listName)
        .map(x => Json.obj("data" -> x))
        .map(Ok(_))
        .getOrElse(NotFound)
```